go/vt/vtgate/planbuilder/operators/route.go

```diff
@@ -717,35 +722,71 @@ func (r *Route) planIsExpr(ctx *plancontext.PlanningContext, node *sqlparser.IsE
 	return r.haveMatchingVindex(ctx, node, vdValue, column, val, opcodeF, justTheVindex)
 }
 
-func createRoute(ctx *plancontext.PlanningContext, table *QueryTable, solves semantics.TableSet) (*Route, error) {
-	if table.IsInfSchema {
-		return createInfSchemaRoute(ctx, table)
+// createRoute returns either an information_schema route, or else consults the
+// VSchema to find a suitable table, and then creates a route from that.
+func createRoute(
+	ctx *plancontext.PlanningContext,
+	queryTable *QueryTable,
```

Comment:
Renamed from `table` to `queryTable`. Thought this would help disambiguate between `vschemaTable`.